docs/reference/pages/panels.rst

```diff
@@ -32,6 +32,20 @@ FieldPanel
 
         This allows you to override the heading for the panel, which will otherwise be set automatically using the form field's label (taken in turn from a model field's ``verbose_name``).
 
+
+.. topic:: Collapsing Panels to save space
+
+    By default, ``FieldPanel`` s are expanded and not collapsible. Adding ``collapsible`` to ``classname`` will enable the collapse control. Adding both ``collapsible`` and ``collapsed`` to the ``classname`` parameter will load the editor page with the ``FieldPanel`` collapsed under its heading. These classes can also be applied to other panel types including ``MultiFieldPanel``, ``InlinePanel`` and ``FieldRowPanel``.
```

Comment:
Good point. I had to go back and test this a bit further. It seems to depend a bit where you are using FieldRowPanel.
If you use FieldRowPanel (or almost any Panel) as the direct child of ObjectList then it can have a heading. The headings are actually output by the ObjectList template, not the Panel's themselves. The collapsible only works if you set a `heading`.

I'll try to update that docs and make that more clear.

Reply:
@rjmackey Rather than explain every nuance, I still feel it would be sensible to only promote use of this for MultiFieldPanel and InlinePanel, as that's where it is most useful/predictable. I think we can safely class collapsing of other panels as an edge-case, and if a developer wants to try that, it's on them to figure out the nuance. 

Another issue with promoting the use of the class everywhere is: It ties us to it forever. We may want to refactor in future to use BEM class naming, or perhaps move to data attributes for controlling this behaviour instead. When we document something, in a sense, it is a promise of long-term support. I would prefer for us not to have to worry about having to preserve collapsible behaviour anywhere.

Reply:
That makes sense. InlinePanel and MultiFieldPanel are the only fields likely to be tall enough for collapsing to be useful.

However it's worth pointing out that the caveats apply to those panels too: ie. they have to be the direct child of ObjectList and must have a heading. I think it's worth making that a little clearer in the docs, even when only documenting it's use with MultiFieldPanel and InlinePanel

Reply:
I've updated the docs to only mention collapsing on the InlinePanel and MultiFieldPanel sections, and added a note about needing to define a heading. Hopefully that solves this.